devicetypes/smartthings/zigbee-thermostat.src/zigbee-thermostat.groovy

```diff
@@ -152,17 +152,17 @@ private parseAttrMessage(description) {
 			if (it.attribute == LOCAL_TEMPERATURE) {
 				log.debug "TEMP"
 				map.name = "temperature"
-				map.value = getTemperature(it.value)
+				map.value = Math.round(getTemperature(it.value))
```

Comment:
The Math.round would probably be best on line 344 in `getTemperature()`

Reply:
agreed. updated